hack/build-cross.sh

```diff
@@ -28,6 +28,7 @@ KUBE_BUILD_PLATFORMS=("${KUBE_SERVER_PLATFORMS[@]}")
 kube::golang::build_binaries "${KUBE_SERVER_TARGETS[@]}"
 
 KUBE_BUILD_PLATFORMS=("${KUBE_CLIENT_PLATFORMS[@]}")
+KUBE_STATIC_OVERRIDES=("kubectl")
```

Comment:
Looks like ```/examples/kubectl-container/Makefile``` sets ```KUBE_STATIC_OVERRIDES``` to ```"kubectl"```.

@lavalamp how would you recommend executing and referencing it in jenkins e2e? As part of the e2e test itself before uploading into ```netexec```?